oidc-client-app-samples/android-client-app-sample/app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,12 @@
+<resources>
```

Comment:
Missing license header.

Reply:
Fixed in PR: https://github.com/wso2/samples-is/pull/179